contrib/opencensus-ext-azure/README.rst

```diff
@@ -118,7 +118,7 @@ Using the Metrics exporter
 
             mmap.measure_int_put(CARROTS_MEASURE, 1000)
             mmap.record(tmap)
-            time.sleep(10)
+            time.sleep(60)
```

Comment:
If the application exits without sleeping, what's the expected result?
What's our recommendation to the users?

Reply:
It is possible that an application will exit before the exporter thread has a chance to exporter the metrics based on exporter_interval.

It is recommended for the user to keep their applications running, or at least long enough for the exporter to meet the interval.

For applications that terminate before the default interval (15s), it is fine because we only have up to one minute of granularity for the ingestion service (soon to be 30s), so the data seen prior to one minute does not have much value.

Reply:
> It is recommended for the user to keep their applications running, or at least long enough for the exporter to meet the interval.

This is vague, what does "long enough" mean? We probably want to document it.

Reply:
The minimum requirement, we don't want to tell the user to "pick some lucky number, wait and pray".

Reply:
Right. Pit of success.